### PR TITLE
Javadocs

### PR DESCRIPTION
**Pull request by SsopoWolox**
Summary

Complete javadoc for all project
Completed build module training
Completed run module training
Completed versioning project for this version

Trello Card
https://trello.com/c/rGYpTlBq/47-javadoc

### CODE REVIEW

**Comment by martinremis** [comment]
Target this PR to #3 and do not merge it until #3 is merged to main.
After merging the base branch, rebase this PR to main.

**Review by jeymarsonwolox — APPROVED** [review]

**Pull request merged**
